tasks/deconst_assets.js

```diff
@@ -80,20 +80,28 @@ module.exports = function(grunt) {
                 headers: {
                     'Authorization': 'deconst ' + ConfigService.get('key')
                 },
-                formData: formData
-            }, function (error, response, body) {
+                formData: formData,
+                timeout: 60000
+            }, function (err, response, body) {
+                if (err) {
+                    grunt.fail.fatal(err);
+                }
+
                 // API key is provided but invalid
                 if (response && response.statusCode === 401) {
                     grunt.fail.fatal("API Key Invalid");
                 }
+                else if (response && response.statusCode !== 200) {
+                    grunt.fail.fatal("Unknown server response");
+                }
 
                 var jsonBody;
 
                 try {
                     jsonBody = JSON.parse(body);
                 }
                 catch (e) {
-                    console.warn(error);
+                    console.warn(err);
```

Comment:
I think `err` would always be null here because of the call to `grunt.fail.fatal` on line 87 above.